Azure/azure-cli
# [Network] Onboard hdinsight private link2 network cmdlets

**Description**

Onboard HDInsight service to network private-link-resource and private-endpoint-connection cmdlets
**Testing Guide**

run az network private-link-resourcce list -h  the parameter --type should contain Microsoft.HDInsight/clusters
az network private-link-resource list --id should work for Microsoft.HDInsight cluster resource id
the same with az network private-endpoint-connection

**History Notes**


[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
